extra/hip_wrapper.py

```diff
@@ -505,3 +680,9 @@ def hiprtcGetCode(prog):
     return e_code
 except:
   if DEBUG >= 1: print("WARNING: libamdhip64.so or libhiprtc.so not found. HIP support will not work.")
```

Comment:
Was there a reason this was only a warning before?

Reply:
Oh thanks, missed that after rebase.